src/build/build.ts

```diff
@@ -82,66 +82,45 @@ export async function build(
           .pipe(gulpif(/\.html$/, new HTMLOptimizeStream(optimizeOptions.html)))
           .pipe(polymerProject.rejoinHtml());
 
-  let buildStream = mergeStream(sourcesStream, depsStream);
-
-  let unbundledPhase = forkStream(buildStream)
-                           .once(
-                               'data',
-                               () => {
-                                 logger.info('Generating build/unbundled...');
-                               })
-                           .pipe(gulpif(
-                               options.insertDependencyLinks || false,
-                               new PrefetchTransform(polymerProject)))
-                           .pipe(dest(unbundledBuildDirectory));
-
-  let bundledPhase = forkStream(buildStream)
-                         .once(
-                             'data',
-                             () => {
-                               logger.info('Generating build/bundled...');
-                             })
-                         .pipe(polymerProject.bundler)
-                         .pipe(dest(bundledBuildDirectory));
+  let buildStream =
+      <NodeJS.ReadableStream>mergeStream(sourcesStream, depsStream)
+          .once('data', () => {
+            logger.debug('Analyzing build dependencies...');
+          });
+
+  if (options.bundle) {
+    buildStream = buildStream.pipe(polymerProject.bundler);
+  } else {
```

Comment:
agreed, made that change in the next PR (https://github.com/Polymer/polymer-cli/pull/510/files#diff-559c8e16c25ba5b201e80baf4b8fc080R90) Just wanted to match current behavior at this point while I was working through.